package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bulma_scss",
-  "version": "0.7.2-rc.0",
+  "version": "0.8.2",
```

Comment:
@illogic-al Please use an rc version here, if this needs to be updated at all, we would have to go to 0.8.3 which would prevent this from lining up with bulma.

Reply:
There are no rc versions. Do you want me to just modify the version number manually?

Reply:
I don't think you understand.  Just curious, have you ever released a package to npm before?  What I am referencing is you updated the version of this package to the same version as the bulma package.  While this is fine, if there are any mistakes or issues in this release, we would then need to update this package to 0.8.3 to re-release.  This would cause this package to not line up to the current bulma and even worse, when bulma comes out with 0.8.3, we cannot use that to update.

Again, the diff here shows what I mean, we were on 0.7.2-rc.0 (which means 0.7.2 of Bulma) but it is a rc (release candidate).  Once released, if we decided that everything was perfect, we could then release the rc as an actual version, but haven't really felt the need to do so as rc's can be used just like other releases.

Reply:
Nope, haven't ever released my own package to npm till yesterday to test my changes.
Thanks for the explanation. Confusion resolved. 
I'll make these two changes requested post-haste!

Reply:
OK. Changes have been pushed.